lib/hook.js

```diff
@@ -139,6 +149,22 @@ function unhookRequire() {
 function hookCreateScript(matcher, transformer, opts) {
     opts = opts || {};
     var fn = transformFn(matcher, transformer, opts.verbose);
+
+    var orgRunInContext = originalScript.prototype.runInContext,
```

Comment:
Move these assignments to top-level variables so that they are initialized early - otherwise by the time someone calls `hookCreateScript` - the original values could have been re-assigned.